Userland/Libraries/LibGL/GLColor.cpp

```diff
@@ -12,3 +12,23 @@ void glColor3f(GLfloat r, GLfloat g, GLfloat b)
 {
     g_gl_context->gl_color(r, g, b, 1.0);
 }
+
+void glColor4f(GLfloat r, GLfloat g, GLfloat b, GLfloat a)
+{
+    g_gl_context->gl_color(r, g, b, a);
+}
+
+void glColor4fv(const GLfloat* v)
+{
+    g_gl_context->gl_color(v[0], v[1], v[2], v[3]);
+}
+
+void glColor4ub(GLubyte r, GLubyte g, GLubyte b, GLubyte a)
+{
+    g_gl_context->gl_color(r, g, b, a);
+}
+
+void glColor4ubv(const GLubyte* v)
+{
+    g_gl_context->gl_color(v[0], v[1], v[2], v[3]);
```

Comment:
Shouldn't this divide the byte by 255.f

Reply:
I didn't notice this in the docs, lemme lookie again

Reply:
Ah yeah, it's written like "blah blah are mapped to..." in the middle of other stuff. oops.